scripts/test/runtests.ps1

```diff
@@ -5,65 +5,79 @@
 
 . "$PSScriptRoot\..\common\_common.ps1"
 
-$TestBinRoot = "$RepoRoot\artifacts\tests"
+$failCount = 0
+
+# Capture PATH for later
+$StartPath = $env:PATH
```

Comment:
Lets keep runtests scripts agnostic of which dotnet tools it is testing. This way it can be reused for testing stage2 or MSI/Debian installations.

Let the calling scripts setup the dotnet tools to be tested.
/cc: @cdmihai